src/main/scala/com/monsanto/arch/cloudformation/model/resource/Events.scala

```diff
@@ -0,0 +1,24 @@
+package com.monsanto.arch.cloudformation.model.resource
+
+import com.monsanto.arch.cloudformation.model._
+import spray.json.{DefaultJsonProtocol, JsonFormat}
+
+case class `AWS::Events::Rule`(name: String,
+                               ScheduleExpression: Token[String],
+                               Targets: Seq[Option[RuleTarget]],
+                               override val Condition: Option[ConditionRef] = None
+                              ) extends Resource[`AWS::Events::Rule`] {
+
```

Comment:
Any reason not to include the remaining parameters from http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-events-rule.html ?